Common/cpp/LayoutAnimations/LayoutAnimationsManager.cpp

```diff
@@ -73,4 +109,23 @@ void LayoutAnimationsManager::startLayoutAnimation(
       config->getJSValue(rt));
 }
 
+int LayoutAnimationsManager::findTheOtherForSharedTransition(int tag) {
+  int theOtherTag = -1;
+  for (auto const &[key, group] : sharedTransitionGroups_) {
+    bool isCurrentGroupMember = false;
+    int indexInGroup;
```

Comment:
same as above, use `std::remove`

Reply:
But this doesn't remove anything? 🤔 
Still, you can replace the `for`-loop with `std::find`